ozaria/site/components/interactive/PageInteractive/draggableStatementCompletion/index.vue

```diff
@@ -72,58 +82,43 @@
         />
       </div>
 
-      <div class="art-container">
```

Comment:
Is the art container not required? Or punted for a future PR?

Reply:
The title bar / trigger art are handled by the `BaseInteractiveLayout` component now.